nikola/plugins/compile/rest/__init__.py

```diff
@@ -63,16 +70,19 @@ def compile_html_string(self, data, source_path=None, is_two_file=True):
             add_ln = len(m_data.splitlines()) + 1
 
         default_template_path = os.path.join(os.path.dirname(__file__), 'template.txt')
+        settings_overrides = {
+            'initial_header_level': 1,
+            'record_dependencies': True,
+            'stylesheet_path': None,
+            'link_stylesheet': True,
+            'syntax_highlight': 'short',
+            'math_output': 'mathjax',
+            'template': default_template_path,
+            'language_code': LocaleBorg().current_lang,
```

Comment:
There will be warnings if a language is unsupported by docutils, but I guess we can live with that. LGTM, merge at will.